content/people/index.md

```diff
@@ -377,7 +390,6 @@ people:
   lastName: Lakdawala
   image: tarannum-lakdawala.jpeg
   affiliation: CMU/CSD
-  note: Contributor, June-December 2021
```

Comment:
Slight preference for changing the field name to something that will be ignored by the frontend instead of deleting this information. For example:

```
  future_note: Contributor, June-December 2021, September 2022-TBD
```

([i tested this out and it works](https://github.com/cmu-delphi/www-main/pull/769))

Reply:
Great suggestion. Agree.